testing/acceptance_test.py

```diff
@@ -896,26 +897,41 @@ def test_has_plugin(self, request):
 class TestDurations:
     source = """
         import time
-        frag = 0.002
+        frag = 0.002             # 2 ms
         def test_something():
             pass
         def test_2():
-            time.sleep(frag*5)
+            time.sleep(frag*5)   # 10 ms: on windows might sleep < 0.05s
```

Comment:
How odd. I'd accept that it might sleep *more* then the time given, but definitely not *less*...

Reply:
See https://stackoverflow.com/a/15967564/7262247

Reply:
A very interesting mod would be to replace `time.sleep()` is the tests with the following function:

```python
def my_sleep(duration, get_now=perf_counter):
    now = get_now()
    end = now + duration
    while now < end:
        now = get_now()
```

(from this post: https://stackoverflow.com/a/60185893/7262247)

Do you think that this is a good idea ? If would remove all the platform-dependent mods in the tests asserts.

Reply:
Hmm busy loops are not great at all, but might be fine for small enough duration. Another way might be to sleep and then sleep again the remainder if it finished before it should have.